crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -289,6 +289,28 @@ fn handle_enclosed_comment<'a>(
             }
         }
         AnyNodeRef::FString(fstring) => CommentPlacement::dangling(fstring, comment),
+        AnyNodeRef::FStringExpressionElement(_) => {
+            // Handle comments after the format specifier (should be rare):
+            //
+            // ```python
+            // f"literal {
+            //     expr:.3f
+            //     # comment
+            // }"
```

Comment:
Ideally comments always get associated with the outer-most node, which `FStringFormatSpec` wouldn't be. 

Maybe change the implementation to always associate it with the outermost element (I'm once more confused by the name and don't know what AST nodes they are :confused: )

Reply:
> Ideally comments always get associated with the outer-most node

I see, this actually makes more sense.

Reply:
For future reference the reason I was thinking to attach it to the format spec is because the format spec is optional which means the comment only occurs if format spec is present.

But I still think attaching it as a trailing comment to the expression element makes more sense. I'm going forward with that.